README.md

```diff
@@ -29,6 +34,37 @@ pip install -e ".[fms-accel]"
 ```
 `fms-acceleration` is a collection of plugins that packages that accelerate fine-tuning / training of large models, as part of the `fms-hf-tuning` suite. For more details on see [this section below](#fms-acceleration).
 
+
+### Using versions of python dependencies that are known to be compatible with fms-hf-tuning
+
+This repository uses [`poetry`](https://python-poetry.org/) for reproducible builds. You can use poetry to create the same virtual environment that we use for our CI/CD tests like so:
+
+```
+python -m venv venv
+. ./venv/bin/activate
+pip install poetry
```

Comment:
Maybe remove the `pip install poetry`. Instead list poetry as a required packages along with the git, python, pip, in the [Set up your dev environment](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/CONTRIBUTING.md#set-up-your-dev-environment) and just hpyerlink to the official installation doc maybe highlighting `Poetry should always be installed in a dedicated virtual environment to isolate it from the rest of your system`